Resources/Private/JavaScript/store.svelte.ts

```
@@ -1,23 +1,105 @@
-export const focuspoints = writable([]);
+export const imageMeta = writable<{ w: number; h: number } | null>(null);
 
-export const initStores = (hiddenInput, wizardConfig, hiddenElement) => {
+type Focuspoint = {[K in string]: string} & {
+  __shape: ShapeType;
+  __data: any;
+}
+
+type ShapeConfig = {
+  component: Function;
+  constructor(config: WizardConfig): object;
```

Biome (2.1.2)

[error] Don't use the constructor method in type aliases.

Type aliases simply rename types. They don't execute code, so a constructoris misleading.

(lint/suspicious/noMisleadingInstantiator)